tools/deployment/test_torchserver.py

```diff
@@ -21,52 +26,94 @@ def parse_args():
         '--device', default='cuda:0', help='Device used for inference')
     parser.add_argument(
         '--score-thr', type=float, default=0.5, help='bbox score threshold')
+    parser.add_argument(
+        '--workdir',
```

Comment:
--workdir --> --work-dir to be consistent with existing arguments in other scripts.